protocol/test/utils/account.ts

```diff
@@ -0,0 +1,8 @@
+import { network } from 'hardhat';
```

Comment:
given it's only one function, I would suggest adding to `helpers.ts`

Reply:
ok